### PR TITLE
Post conf blog

### PR DESCRIPTION
**Pull request by helenalt**
Please review. I will rename this file with the December date once we know when we can post it (when video links are available).

### CODE REVIEW

File: _posts/2017-11-30-thanks-bazelcon2017.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: posts
+title: Thank you for BazelCon 2017
+---
+
+We are truly thankful to our community for making our first annual [**Bazel Conference**](https://sites.google.com/corp/bazel.build/conference2017) a success! Talk videos are now published on YouTube and you can access them [here](link).
+
+Your feedback reflected high level of satisfaction, as we had topics of interest for everyone:
+* Bazel usage ang migration stories from TensorFlow, Kubernetes, SpaceX, Pinterest, Wix, Stripe, DataBricks, and Dropbox.
```
Comment (edbaunton) [inline _posts/2017-11-30-thanks-bazelcon2017.md]: Ang -> and

File: _posts/2017-11-30-thanks-bazelcon2017.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* 20 informative talks
+* 14 hours of hands-on debugging and Q&A during Office Hours
+
+What we heard from you:
+* Bazel Query is great!
+* Python is used broader than we thoght, and Bazel support is important.
+* Reproducible builds are important especially when you're flying rockets, building autonomous vehicles, delivering media, and calculating financial transactions, leading to wider Bazel adoption by these communities.
+* Documentation is often hard to find, and is either too basic or too advanced.
+* Bazel's Parallelism is impressive: much faster than Maven.
+* IDE integration is important, particularly with Visual Studio and CLion.
```
Comment (edbaunton) [inline _posts/2017-11-30-thanks-bazelcon2017.md]: And xcode?

File: _posts/2017-11-30-thanks-bazelcon2017.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Build Event Protocol enables many options for internal visualization of events.
+* Aspects are a powerful tool.
+
+What we could do next:
+* Work on better documentation and training, particularly for intermetiate-level.
```
Comment (edbaunton) [inline _posts/2017-11-30-thanks-bazelcon2017.md]: Intermediate

File: _posts/2017-11-30-thanks-bazelcon2017.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* Prioritize IDE intregration.
```
Comment (edbaunton) [inline _posts/2017-11-30-thanks-bazelcon2017.md]: Integration

**Comment by edbaunton** [comment]
Nit-picks

File: _posts/2017-11-30-thanks-bazelcon2017.md (hunk 1 above)
Comment (helenalt) [reply _posts/2017-11-30-thanks-bazelcon2017.md]: ack

File: _posts/2017-11-30-thanks-bazelcon2017.md (hunk 2 above)
Comment (helenalt) [reply _posts/2017-11-30-thanks-bazelcon2017.md]: ah yes!

File: _posts/2017-11-30-thanks-bazelcon2017.md (hunk 3 above)
Comment (helenalt) [reply _posts/2017-11-30-thanks-bazelcon2017.md]: ack

**Pull request merged**